package/Dockerfile

```diff
@@ -29,14 +29,14 @@ COPY --from=intermediate /kube-bench/cfg /etc/kube-bench/cfg/
 
 COPY package/cfg/ /etc/kube-bench/cfg/
 COPY package/run.sh \
-     package/run_sonobuoy_plugin.sh \
-     bin/kb-summarizer \
-     package/helper_scripts/check_files_permissions.sh \
-     package/helper_scripts/check_files_owner_in_dir.sh \
-     package/helper_scripts/check_encryption_provider_config.sh \
-     package/helper_scripts/check_for_network_policies.sh \
-     package/helper_scripts/check_for_default_sa.sh \
-     package/helper_scripts/check_for_default_ns.sh \
-     /usr/bin/
+    package/run_sonobuoy_plugin.sh \
+    bin/kb-summarizer \
+    package/helper_scripts/check_files_permissions.sh \
+    package/helper_scripts/check_files_owner_in_dir.sh \
+    package/helper_scripts/check_encryption_provider_config.sh \
+    package/helper_scripts/check_for_network_policies.sh \
+    package/helper_scripts/check_for_default_sa.sh \
+    package/helper_scripts/check_for_default_ns.sh \
+    /usr/bin/
```

Comment:
Indentation seems to be missing one space? It looked correct earlier

Reply:
Indentation was 5 and I updated it to the standard 4.